meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -484,6 +484,8 @@ EXTRA_IMAGEDEPENDS:append:stm32mp15-eval = "flashlayouts-stm32mp1"
 LMP_FLASHLAYOUT:stm32mp15-eval = "FlashLayout_stm32mp1-optee.tsv"
 LMP_FLASHLAYOUT_BOARD_NAME:stm32mp15-eval = "stm32mp157c-ev1"
 LMP_BOOT_FIRMWARE_FILES:stm32mp15-eval = "arm-trusted-firmware/tf-a-stm32mp157c-ev1-emmc.stm32 fip/fip-stm32mp157c-ev1-optee.bin"
+IMAGE_INSTALL:append:stm32mp15-eval = " kernel kernel-devicetree u-boot-base-scr"
+IMAGE_INSTALL:remove:sota:stm32mp15-eval = "kernel kernel-devicetree u-boot-base-scr"
```

Comment:
We shouldn't change IMAGE_INSTALL here, please switch with MACHINE_ESSENTIAL_EXTRA_RDEPENDS instead (and no need to remove when sota is used).

Also use u-boot-default-script instead of u-boot-base-scr.

Reply:
But why should we keep `kernel kernel-devicetree` for sota?

Reply:
I don't think installing them would cause any side effect (and I think kernel is already installed), but if you think it is better to have it not installed, then do a similar append/remove logic but using MACHINE_ESSENTIAL_EXTRA_RDEPENDS instead.